draft-ietf-quic-transport.md

```diff
@@ -5257,7 +5267,8 @@ initial_max_stream_data_uni (0x07):
 initial_max_streams_bidi (0x08):
 
 : The initial maximum bidirectional streams parameter is an integer value that
-  contains the initial maximum number of bidirectional streams the peer is
+  contains the initial maximum number of bidirectional streams the endpoint
+  that receives this transport parameter is
```

Comment:
We were inconsistent about how we described the two endpoints in these paragraphs.  I'm not sure I caught all instances, though, but the "the endpoint that receives this transport parameter" seems like a good clear description.